OrbitService/ProcessServiceImpl.cpp

```diff
@@ -27,3 +30,55 @@ Status ProcessServiceImpl::GetProcessList(ServerContext*,
 
   return Status::OK;
 }
+
+Status ProcessServiceImpl::GetSymbols(ServerContext*,
+                                      const GetSymbolsRequest* request,
+                                      GetSymbolsResponse* response) {
+  const auto& process = process_list_.GetProcess(request->process_id());
+  if (!process) {
+    std::string error_message =
+        absl::StrFormat("The process with pid=%d was not found on remote",
+                        request->process_id());
+    ERROR("%s", error_message);
+    return Status(StatusCode::NOT_FOUND, error_message);
+  }
+
+  process->ListModules();
+  const auto& module = process->GetModuleFromName(request->module_name());
```

Comment:
you will not need these if request contains module_path.. also this is relying on module name being unique which is not always true. you can have /usr/lib/libbase.so and /opt/somedir/libbase.so

Reply:
I used path now everywhere -> done